src/main.rs

```diff
@@ -168,6 +168,7 @@ enum Commands {
         /// Optional path to a new directory in which to save the file(s). If none is specified writes the data to STDOUT.
         #[clap(long, short)]
         out: Option<PathBuf>,
+        /// True to download a single blob, false (default) to download a collection and its children.
```

Comment:
you don't need to call out the default, clap already does this if you use `default_value_t`.  (please don't change the other options, i have an not-yet-pr that cleans this all up for the existing options)